jdaviz/app.py

```diff
@@ -773,9 +773,14 @@ def _get_all_subregions(mask, spec_axis_data):
                     # TODO: this needs to be much simpler; i.e. data units in
                     #  the glue component objects
                     # Cases where there is a single subset
+                    if '_orig_wcs' in value.data.meta:  # Hack for Cubeviz WCS propagation
+                        data_wcs = value.data.meta['_orig_wcs']
```

Comment:
this line currently is uncovered... but since its so critical, I wonder if we should try to add coverage?

Reply:
Agreed, I'll work on that.

Reply:
I wasn't sure what use case will trigger this, so I'll leave the test in Jesse's good hands.